readme.md

```diff
@@ -2,7 +2,7 @@
 
 ## Jest
 
-1) Setup a project on [Buildkite Test Analytics](https://buildkite.com/test-analytics) and note the key
+1) Setup a project on [Buildkite Test Analtics](https://buildkite.com/test-analytics) and note the key
```

Comment:
This typo snuck back in too

Reply:
good to see my typo continiously coming back to haunt me 😢

Reply:
😂💚